backend/test/message.js

```diff
@@ -0,0 +1,169 @@
+const chai = require('chai');
+const server = require('../index');
+const chaiHttp = require('chai-http');
+const chaiJsonSchema = require('chai-json-schema');
+const router = require('../routes/studentRoute');
+const { response } = require('express');
+const { request } = require('chai');
+const { post } = require('../index');
+
+//Assertion style
+chai.should();
+
+chai.use(chaiHttp);
+chai.use(chaiJsonSchema);
+
+describe('message API', () => {
+
+    
+    
+
+    describe('POST /message', () => {
+        
+        let msg = {
+            message: "nice to meet you",
+            sender: "STUDENT",
+            sendDate: "2020-11-10T12:50:00.000Z"
+        }
+
+        beforeEach(async () => {
+            msg.employerId = (await chai.request(server)
+                .get('/employers/last')).body.id
+
+        })
+
+        beforeEach(async () =>{
+            
```

Comment:
I think that we are supposed to add just 1 handler beforeEach, and we can add both calls in the same handler